base/src/main/java/org/aya/concrete/visitor/ExprOps.java

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2020-2022 Yinsen (Tesla) Zhang.
+// Use of this source code is governed by the MIT license that can be found in the LICENSE.md file.
+package org.aya.concrete.visitor;
+
+import org.aya.concrete.Expr;
+import org.jetbrains.annotations.NotNull;
+
+public interface ExprOps extends ExprView {
+  @NotNull ExprView view();
+  @Override default @NotNull Expr initial() {
+    return view().initial();
+  }
+
+  @Override default Expr pre(Expr expr) {
```

Comment:
It's better to note the parameter and return type with `@NotNull` or `@Nullable`